staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store_test.go

```diff
@@ -2042,6 +2042,9 @@ func TestStoreDeleteCollection(t *testing.T) {
 	storeWithCounter := &storageWithCounter{Interface: registry.Storage.Storage}
 	registry.Storage.Storage = storeWithCounter
 	deleteCollectionPageSize = 2
+	defer func() {
+		deleteCollectionPageSize = int64(10000)
+	}()
```

Comment:
Great finding. Instead of setting 10000 value here, can you please restore to the value to which it was set before?
So sth like:
```
originalDeleteCollectionPageSize = deleteCollectionPageSize
deleteCollectionPageSize = 2
defer func() {
  deleteCollectionPageSize = originalDeleteCollectionPageSize
}()
```

Reply:
all right!